src/MSBuild/XMake.cs

```diff
@@ -127,7 +127,9 @@ public enum ExitType
         /// <summary>
         /// Static constructor
         /// </summary>
+#pragma warning disable CA1810 // Initialize reference type static fields inline
         static MSBuildApp()
+#pragma warning restore CA1810 // Initialize reference type static fields inline
```

Comment:
nit: I think I'd slightly prefer putting the restore below the end of this function, but I don't care too much.